api/ccapi/include/functions.h

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 SeungBaek Hong <sb92.hong@samsung.com>
+ *
+ * @file   functions.h
+ * @date   15 March 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author SeungBaek Hong <sb92.hong@samsung.com>
+ * @bug	   No known bugs except for NYI items
+ *
+ * @note This is experimental API and not stable.
+ */
+
+#ifndef __ML_TRAIN_TENSOR_H__
+#define __ML_TRAIN_TENSOR_H__
```

Comment:
this conflicts with `tensor_api.h`. 
https://github.com/nnstreamer/nntrainer/blob/7fb71e2e0f0f514ef72a0294be3ced0a5999beee/api/ccapi/include/tensor_api.h#L15-L16

maybe change to something else
```suggestion
#ifndef __ML_TRAIN_FUNCTIONS_H__
#define __ML_TRAIN_FUNCTIONS_H__
```

Reply:
I modified it. Thanks :)